src/AlixBa/Addic7edSubtitles/Helpers/Episode.php

```diff
@@ -51,8 +51,8 @@ public function __construct($episodeFilename)
     {
         preg_match($this->pattern, $episodeFilename, $matches);
 
-        $this->showName          = $matches['showname'];
-        $this->sanitizedShowName = self::sanitizeShowName($matches['showname']);
+        $this->showName          = self::capitalizeShowName($matches['showname']);

```

Comment:
I hope i understood you correctly.

Added another patch to this pull request; in it i just remove the sanitizedShowName property from Episode, and use showName all over the place instead. Also merge the two "sanitize" functions into one; sanitizeAndCapitalizeShowName()  behaviour should be identical to before.